data/curriculum/1.json

```
@@ -0,0 +1,92 @@
+{
+    "id": 1,
```

And then this should be the start of a new object
```suggestion
    {
      "id": 1,
```